backend/api/Startup.cs

```diff
@@ -98,13 +98,8 @@ public void ConfigureServices(IServiceCollection services)
                         OnAuthenticationFailed = context =>
                         {
                             context.NoResult();
-                            context.Response.StatusCode = 500;
-                            context.Response.ContentType = "text/plain";
-                            if (Environment.IsDevelopment())
-                            {
-                                return context.Response.WriteAsync(context.Exception.ToString());
-                            }
-                            return context.Response.WriteAsync("An error occurred processing your authentication.");
+                            context.Response.StatusCode = 401;
```

Comment:
Authentication errors will now use the error handling middleware and return 401.